examples/using-jss/package.json

```diff
@@ -2,11 +2,14 @@
   "name": "gatsby-example-using-jss",
   "private": true,
   "description": "Gatsby example site using the JSS plugin",
-  "version": "1.0.0",
+  "version": "2.0.0",
```

Comment:
do we really want to bump this version?

Reply:
I took the other PRs as reference. https://github.com/gatsbyjs/gatsby/pull/5717 Check this out.

Reply:
Hmm I don't think it really make sense for the examples to have versions at all, as they're not published anywhere. Some actually don't have a version, but most have stayed as v1. I've reverted this, but maybe we need to take a closer look at the example versions later.

Reply:
True, I agree, I was assuming, we were following the convention will shifting to Gatsby v2 and so went forward with the changes. And so yes, later we should have a look at all the examples versions.